admin/system/insert_berita.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    //get data
+    $judul_berita = $_POST["judul_berita"];
+    $kategori = $_POST["kategori"];
+    $id = uniqid();
+    mkdir("../../news/".$id);
+
+    if(isset($_FILES["gambar1"])){
+        mkdir("../../news/".$id."/gambar");
```

Comment:
Codacy found an issue: [The use of function mkdir() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)